components/jvm-build-service/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+resources:
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/kcp?ref=bc1ecfaf82f1edaef5b4d80e53d093e371048fb7
```

Comment:
Cannot be evaluated:
```
# kustomize build https://github.com/redhat-appstudio/jvm-build-service/deploy/kcp?ref=bc1ecfaf82f1edaef5b4d80e53d093e371048fb7
Error: accumulating resources: accumulation err='accumulating resources from '../kcp-manager': '/tmp/kustomize-788111474/deploy/kcp-manager' must resolve to a file': recursed accumulation of path '/tmp/kustomize-788111474/deploy/kcp-manager': add operation does not apply: doc is missing path: "/spec/template/spec/containers/2/args/-": missing value

```

Reply:
yeah expected - the commit level is out of date, and the PR has not merged yet

for now, I'm just depicting future intent with this change.

I should have  conveyed that in a comment.

this will be corrected once the jvm-build-service PR has merged, which I'll do once I've been able to successfully bring up on kcp locally; that will most likely coincide with my switching this out of draft status.